src/redux/detailReducer/detailSlice.js

```diff
@@ -0,0 +1,44 @@
+import { createAsyncThunk, createSlice } from '@reduxjs/toolkit';
+import axios from 'axios';
+// import { breedURL } from '../../components/API';
```

Comment:
Great job @Rafiduzzaman 🤝

- [ ] Kindly remove commented code from your project as this is regarded as bad practice.